src/services/doppler-legacy-client.ts

```diff
@@ -452,4 +471,14 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
       };
     }
   }
+
+  public async getBannerData(lang: string, type: string, page: string) {
+    const response = await this.axios.get(
+      `https://qa.fromdoppler.com/wp-json/doppler2019/v1/getbanner?filter[lang]=${lang}&filter[type]=${type}&filter[page]=${page}`,
```

Comment:
I think that you will need to create a new client, what do you think about _DopplerSiteClient_?

Reply:
done